tests/test_volume.py

```diff
@@ -476,3 +477,48 @@ def test_asnumpy_readonly():
     # Attempt assignment
     with raises(ValueError, match=r".*destination is read-only.*"):
         vw[0, 0, 0, 0] = 123
+
+
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def testProjectBroadcast(dtype):
+    L = 32
+
+    # Create stack of 3 Volumes.
+    n_vols = 3
+    vols = AsymmetricVolume(L=L, C=n_vols, dtype=dtype).generate()
+
+    # Create a singleton and stack of Rotations.
+    rot = Rotation.about_axis("z", np.pi / 6, dtype=dtype)
+    rots = Rotation.about_axis("z", [np.pi / 4, np.pi / 3, np.pi / 2], dtype=dtype)
+
+    # Test mask.
+    mask = grid_2d(L)["r"] < 1
+
+    # Broadcast Volume stack with singleton Rotation and compare with individually generated projections.
+    projs_3_1 = vols.project(rot).asnumpy()
+    for i in range(n_vols):
+        proj_i = vols[i].project(rot).asnumpy()
+        assert np.allclose(
+            projs_3_1[i][mask],
+            proj_i[0][mask],
+            atol=utest_tolerance(dtype),
+        )
+
+    assert projs_3_1.shape[0] == n_vols
+
+    # Broadcast Volume stack with Rotation stack of same size and compare with individually generated projections.
+    projs_3_3 = vols.project(rots).asnumpy()
+    for i in range(n_vols):
+        proj_i = vols[i].project(rots[i]).asnumpy()
+        assert np.allclose(
+            projs_3_3[i][mask],
+            proj_i[0][mask],
+            atol=utest_tolerance(dtype),
+        )
+
+    assert projs_3_3.shape[0] == n_vols
+
```

Comment:
Lets just add a comment that the single volume to multiple rotation case is covered by `testProject`.  I had to go look for it to check it was covered 😇 . One day when this gets refactored it might go here...

Reply:
Yep, should have mentioned that. Thanks! Done